game/debriefing.py

```diff
@@ -121,7 +117,10 @@ def from_json(cls, data: Dict[str, Any]) -> StateData:
             killed_aircraft=data["killed_aircrafts"],
             # Airfields emit a new "dead" event every time a bomb is dropped on
             # them when they've already dead. Dedup.
-            killed_ground_units=list(set(data["killed_ground_units"])),
+            #
+            # Also normalize dead map objects (which are ints) to strings. The unit map
+            # only stores strings.
+            killed_ground_units=list({str(u) for u in data["killed_ground_units"]}),
```

Comment:
Would this cause things to break if this weren't done?  This is the only thing that I see different from my attempts.

Reply:
The debriefing wasn't finding the scenery deaths before because it was giving an `int` key to the unit map, which didn't match the unit map's string keys.

Reply:
Ah I see, thanks.